cinema/views.py

```diff
@@ -0,0 +1,41 @@
+from django.http import HttpRequest
+from django.shortcuts import render, get_object_or_404
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema_service.serializers import MovieSerializer
+
+
+def index(request: HttpRequest):
+    return render(request, "base.html")
```

Comment:
Why you need this?

Reply:
Removed, I made a default page for entering